src/foam/box/HTTPBox.js

```diff
@@ -26,8 +26,9 @@ foam.CLASS({
   ],
 
   imports: [
+    'generator? as ctxGenerator',
```

Comment:
For example, I've verified that native JSON parsing performs better when storing a cache in a worker and using `MessagePortBox` to access the DAO on the main thread. I cannot do that unless I can choose `JSONParserGenerator`.